src/inverseRobot.js

```diff
@@ -25,6 +25,16 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const result = {};
+
+  for (const i in robot) {
```

Comment:
Please add more understandable name for "i".

Reply:
agree, use more understandable name for variables